Automattic/jetpack
# AI Client: tweak layout and styles to make AI Control mobile friendly

Fixes https://github.com/Automattic/jetpack/issues/32297

## Proposed changes:

* AI Client: tweak layout and styles to make AI Control mobile friendly

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Use Storybook

```
cd projects/js-packages/storybook/
```

```
pnpm run storybook:dev
```

Take a look at the AI Control story:

* Do not show buttons label

<img width="661" alt="Screenshot 2023-08-09 at 14 54 27" src="https://github.com/Automattic/jetpack/assets/77539/e91b9e76-b0d5-440d-8fd6-f53605d92f6c">


* Confirm the following changes


Size | Input | **after** | **before**
--------|-------|------|------
Small | Empty | <img width="342" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/7d94d6ba-838b-4af9-914b-46e845ce87c8">|<img width="338" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/d4d0539a-26e2-4ecb-9c2b-d6d36edbc976">
Small | Not Empty | <img width="339" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/c18b13f0-6709-468f-904e-94a673d1f817">|<img width="334" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/b3bc499f-3b14-47d2-8620-de69feb038cb">
Medium | Empty | <img width="436" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/a0dc28db-dab5-4db9-9217-2502c5072c37">|<img width="425" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/42c97ad2-261b-4153-8540-58a37de36ec3">
Medium | Not empty | <img width="432" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/1d644a3f-dab8-4d54-b198-9012fd093aae">|<img width="427" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/faa796fb-051b-4815-b3d4-8dfd572d3ddc">

* Show button labels

**before**

<img width="540" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/f923bb22-dd94-48b9-8b4c-70e2e542d5b3">

**after**
<img width="533" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/de7c8d5e-d198-4ac9-b254-de556ae5e8fb">
